app/common/helpers/collections.py

```diff
@@ -137,6 +137,12 @@ def get_first_question_for_form(self, form: "Form") -> Optional["Question"]:
             return questions[0]
         return None
 
+    def get_last_question_for_form(self, form: "Form") -> Optional["Question"]:
```

Comment:
In some previous thinking on the pattern something I didn't hate was using `get_next_question` and `get_previous_question` to get the first and last question if you didn't provide a relative question to go from. Interested in thoughts and not even a steer, maybe helpful for having slightly less methods to learn but there's always a trade off!